lntest/harness_node.go

```diff
@@ -317,41 +317,50 @@ type HarnessNode struct {
 	// node and the outpoint.
 	policyUpdates policyUpdateMap
 
+	// conn is the underlying connection to the grpc endpoint of the node.
+	conn *grpc.ClientConn
```

Comment:
Move this to `RPCClients` struct as well?

Reply:
Done!